media/js/src/simulationOne/simulationOneQuiz.jsx

```diff
@@ -15,6 +15,8 @@ export const SimulationOneQuiz = ({
         choice => completedChoices.includes(choice));
 
     const handleChoiceCompletion = () => {
+        document.getElementById('quiz-1')
```

Comment:
This can be removed here. The way the components are displayed it jumps back up anyway.